OMSEventWriter.cpp

```diff
@@ -206,6 +215,10 @@ ssize_t OMSEventWriter::WriteEvent(const Event& event, IWriter* writer)
     end_object(); // Event
     end_array(); // Message
 
+    if (records == 0) {
+        reset();
```

Comment:
There's a reset() at the beginning of the function, so this one isn't necessary.